drivers/SmartThings/zwave-thermostat/src/init.lua

```diff
@@ -37,6 +48,10 @@ local function device_added(driver, device)
     device:is_cc_supported(cc.THERMOSTAT_FAN_MODE) then
     device:send(ThermostatFanMode:SupportedGet({}))
   end
+  if device_supports_thermostat_setpoint(device) then
+    device:send(ThermostatSetpointV3:CapabilitiesGet({setpoint_type = ThermostatSetpoint.setpoint_type.HEATING_1}))
+    device:send(ThermostatSetpointV3:CapabilitiesGet({setpoint_type = ThermostatSetpoint.setpoint_type.COOLING_1}))
```

Comment:
Just curious why `device_supports_thermostat_setpoint` returns true if one or the other of the heating and cooling setpoint capabilities is supported, but here we are getting both. Would it make sense to only get what is supported?

Reply:
That is a good point. We do have some heat-only thermostats.

Reply:
Thanks for pointing out, I didn't think of that case. I adjusted this to be two conditions: Check for support for each heating and cooling setpoints independently before sending the request for the corresponding bounds.